core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1590,9 +1590,10 @@ public Map<String, Map<String, Long>> getTopicRequestsCounts(
       operationTypeCountsMap.put(RequestOperationType.CLAIM.value, assignedToClaimReqs);
       if (RequestMode.MY_APPROVALS == requestMode) {
         // Make sure to remove any requests which the requestor can not approve
+        // Using Integer.toString() as it seems to be the fastest method of changing it.
         Long topicApprovalCount =
             topicRequestsRepo.countRequestorsTopicRequestsGroupByStatusType(
-                teamId, tenantId, requestor, RequestStatus.CREATED.value);
+                teamId, Integer.toString(teamId), tenantId, requestor, RequestStatus.CREATED.value);
```

Comment:
Do we want to have a different teamId in the second argument here

Reply:
I dont think so, the teamId is the requestors teamId and so is retriving the statistics for all topics/connector requests they are able to approve, and that aligns with what we return to the UI in terms of the connectors themselves.